aptos-move/framework/src/natives/randomness.rs

```diff
@@ -55,14 +59,31 @@ pub fn fetch_and_increment_txn_counter(
     _ty_args: Vec<Type>,
     _args: VecDeque<Value>,
 ) -> SafeNativeResult<SmallVec<[Value; 1]>> {
+    if context.gas_feature_version() >= RELEASE_V1_13 {
+        context.charge(RANDOMNESS_FETCH_AND_INCREMENT_BASE)?;
+    }
```

Comment:
@vgao1996 - should every native function charge some base gas, or if function is simple enough - charging of native call is enough?

i.e. randomness had no gas charging before, so I added it here. not sure if that is needed